### ISSUE TITLE
Send contribution later, without deadline

### ISSUE THREAD

**Issue by loma-one** [friendica/friendica#14230]
### Describe the feature you'd like

More and more often, I find myself wanting to create a post but wanting to wait before sending it. For example, because I'm still looking for information, the right picture, a suitable moment.
In principle, I can date a post to the future. However, such posts cannot be called up again to edit them further. 

I would therefore like to be able to start a post today, save it with "Save without sending" and then edit it again in the future.
To avoid having to realise a new save function, the "Save without sending" can be realised with a date far in the future (e.g. 31.12.9999). 

Only 1 1/2 functions are required ;) 
1. reopening a scheduled post. 
2. a button that sets the post to a date in the future.

**Comment by HankG** [comment]
Having a "drafts" feature is something I have thought about as well.

**Comment by tobiasd** [comment]
I'm using the "personal notes" (among real notes) for this. Once the posting is ready, I copy the content over to the post box.

**Comment by loma-one** [comment]
@tobiasd That's how I do it at the moment. I have all the links in the correct form, images are integrated via the gallery, etc. I can check in the preview whether everything fits. However, it would be nice if I could save myself this copying :)

**Comment by annando** [comment]
The easiest way would possibly be to create an option to copy the notes into a regular post. I'm not sure if this is easy to do.

**Comment by loma-one** [comment]
@annando There is such an option in the calendar.

**Comment by loma-one** [comment]
Is reopening a post that is parked under ‘scheduled posts’ a very big challenge? This would fulfil 99% of the requirement.

**Comment by annando** [comment]
Yeah, that's not so easy.
